.github/workflows/ruby.yml

```
@@ -5,14 +5,16 @@ on: [push, pull_request]
 jobs:
   test:
     runs-on: ubuntu-latest
-
+    strategy:
+      matrix:
+        ruby: [ '2.6', '2.7', '3.0' ]
```

I suggest to test all versions declared as supported by gem.

```suggestion
        ruby: [ '2.4', '2.5', '2.6', '2.7', '3.0' ]
```